README.md

```diff
@@ -278,6 +278,124 @@ Specifies the phase to be processed.
 > In the case of a call from a subrequest, `auth_jwt_key_request` cannot
 > be processed. (nested in-memory subrequest)
 
+```
+Syntax: auth_jwt_locked_subs_list_file file;
+Default: -
+Context: http, server, location
+```
+
+Specifies a file for locked subs.
+Parameter value can contain only filepath to json file with objects. Every object should have key(jwt sub) and any additional value, if it needed.
+> Example of config:
+> ```
+  > auth_jwt_locked_subs_list_file /home/aleksey/projects/other/angie/aleksey/revocation/lockeduserslist.json;`
+> ``` 
+
+> Example of file: 
+> ```
+> {
+>   "lockedsub1": {"locked_at": "2023"},
+>   "lockedsub2": {"locked_reason": "bad user"},
+>   "lockedsub3": {"any_other_property": 1}
+> }
+> ```
+
+```
+Syntax: auth_jwt_revocation_kid_list_file file;
+Default: -
+Context: http, server, location
+```
+
+Specifies a file for revocation_kids
+Parameter value can contain only filepath to json file with objects. Every object should have key(jwt header kid) and any additional value, if it needed.
+
+> Example of config:
+> ```
+  > auth_jwt_revocation_kid_list_file /home/aleksey/projects/other/angie/aleksey/revocation/lockeduserslist.json;`
+> ``` 
+
+> Example of file:
+> ```
+> {
+>   "test2kid": {"revocation_reason": "unknown"}
+> }
+
+> ```
+
+**Note:** as we know, kid is OPTIONAL parameter by [rfc7515](https://datatracker.ietf.org/doc/html/rfc7515#page-11), but if you are using auth_jwt_revocation_kid_list_file directive - it means, that kid will grow to **REQUIRED**
+
+```
+Syntax: auth_jwt_require_claim claim_name operator=eq|ne|gt|ge|lt|le|intersect|nintersect|in|nin required_json_value=$variable;
```

Comment:
this notation is mistaken for `auth_jwt_require_claim claim_name operator=eq required_json_value=$variable;` so `auth_jwt_require_claim claim_name operator $variable;` is better.